robot/EDA/tests/browser/settings/relationships_reciprocal.robot

```diff
@@ -0,0 +1,127 @@
+*** Settings ***
+Documentation   Validate reciprocal relationships sub tab in EDA Settings
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/RelationshipsSettingsPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser       AND
+...             Go to EDA settings tab          Relationships      AND
+...             Go to relationships sub tab     Reciprocal Settings     AND
+...             Initialize test data
```

Comment:
You should add this either in Test Setup or Suite setup, but not both

Reply:
I am accessing the suite variable settings count through UI for which I need the system to login and go to settings sub tab. So I had to keep this step in Suite Set Up. But I added the same in test set up too because my other test cases needs these steps as well. For now I can't think of anything else other than duplicating it once. But recently I met with dev's and identified a way to read the same count through API's. I will have to rework this Suite Setup later and change it to API and removing the UI steps is the plan